test/integration/run.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -e
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+echo "Running VPC Resource Controller integration test with the following variables
+KUBE CONFIG: $KUBE_CONFIG_PATH
+CLUSTER_NAME: $CLUSTER_NAME
+REGION: $REGION"
+
+CLUSTER_INFO=$(aws eks describe-cluster --name $CLUSTER_NAME --region $REGION)
+
+VPC_ID=$(echo $CLUSTER_INFO | jq -r '.cluster.resourcesVpcConfig.vpcId')
+SERVICE_ROLE_ARN=$(echo $CLUSTER_INFO | jq -r '.cluster.roleArn')
+ROLE_NAME=${SERVICE_ROLE_ARN##*/}
+ 
+echo "VPC ID: $VPC_ID, Service Role ARN: $SERVICE_ROLE_ARN, Role Name: $ROLE_NAME"
+
+# Set up local resources
+echo "Attaching IAM Policy to Cluster Service Role"
+aws iam attach-role-policy \
+    --policy-arn arn:aws:iam::aws:policy/AmazonEKSVPCResourceController \
+    --role-name "$ROLE_NAME" > /dev/null
+
+echo "Enabling Pod ENI on aws-node"
+kubectl set env daemonset aws-node -n kube-system ENABLE_POD_ENI=true
+
```

Comment:
Do we want to wait for at least 60 seconds to let aws-nodes restart?

Reply:
Given that the Ginkgo suite only tests Pods using SGP. It will create the `Security Group Policy` and pods targeted by that SGP so in the worst case scenario when the `aws-node` doesn't start the Pods will be stuck in Pending State (as aws-node has not yet notified vpc-resource-controller to advertise the branch ENI capacity on the node) and once the aws-node finally restarts the Pod would go into Running state.

In short, the Ginkgo suite will not be impacted if the aws-node is still no started with the new environment variable.

Reply:
So test code in Ginkgo has the mechanism added to wait for pod till Running state?

Reply:
Yes it waits for the Pod to go in Running state and if it doesn't then `-timeout 15m` when executing the tests will stop the timeout the failed test.